src/Cursors.ts

```diff
@@ -48,7 +49,7 @@ export default class Cursors extends EventEmitter<CursorsEventMap> {
     const self = await this.space.members.getSelf();
 
     if (!self) {
-      throw new Error('Must enter a space before setting a cursor update');
+      throw ERR_OUT_OF_SPACE;
```

Comment:
```suggestion
      throw ERR_NOT_ENTERED_SPACE;
```

Reply:
Changed in 6a2e961a3477979133cec5f4d98ca61a65b30ff7